packages/OS400/make-src.sh

```diff
@@ -47,6 +47,13 @@ fi
 get_make_vars Makefile.inc
 
 
+#       Add hugehelp, as it is not included in Makefile.inc.
+if [ "${USE_MANUAL}" = "'USE_MANUAL'" ]
+then
+    CURL_CFILES="${CURL_CFILES} tool_hugehelp.c"
+    CURL_HFILES="${CURL_HFILES} tool_hugehelp.h"
```

Comment:
```suggestion
        CURL_CFILES="${CURL_CFILES} tool_hugehelp.c"
        CURL_HFILES="${CURL_HFILES} tool_hugehelp.h"
```
nit: use 8 spaces for consistency with the rest of file?

Reply:
Done